espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ViewsCache.kt

```diff
@@ -38,3 +39,19 @@ object ViewsCache {
         cache.evictAll()
     }
 }
+
+/** Compose does not have stable instances, use locator again. */
+object ComposeViewCache {
```

Comment:
ComposeViewsCache/ClassicViewsCache

Reply:
changed to ComposeViewsCache/EspressoViewCache